Proctor_Brad/Assignments/selection sort.py

```diff
@@ -0,0 +1,15 @@
+import random
+b = []
+for x in range(0,100):
+    b.append(int(random.random()*10000))
+
+minimum = 0
+for j in range(0, 100):
+    for i in range(minimum, 100):
+        if(b[i] < b[j]):
```

Comment:
I don't think this is a selection sort =X

Try to duplicate ![what this image is doing](https://upload.wikimedia.org/wikipedia/commons/9/94/Selection-Sort-Animation.gif)